test/IISIntegration.FunctionalTests/OutOfProcess/HttpsTest.cs

```diff
@@ -33,7 +33,6 @@ public static TestMatrix TestVariants
             var deploymentParameters = new DeploymentParameters(variant)
             {
                 ApplicationPath = Helpers.GetOutOfProcessTestSitesPath(),
-                ApplicationBaseUriHint = "https://localhost:44394/",
```

Comment:
Does this need to be  "https://localhost:0/" so it gets the scheme hint?

Reply:
For some reason the https://localhost:0 does not work.  The tests will get socket aborted exception when doing it, if I explicitlly call GetNextSSLPort and feed it into the BaseUriHint (which is what should be happening anyways) it works...
 
Any ideas @Tratcher ?

Reply:
Hmm, it should.. https://github.com/aspnet/Hosting/blob/cfab182399e2a22f3e6d87a8875396fced6078c8/src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs#L145-L148
Did you rebase and update deps?

Reply:
Yes it works fine for the non https case in Ntlm for example, so there's something weird with the SSL specifically and the hint.